com.unity.netcode.gameobjects/Samples/ClientNetworkTransform/Scripts/ClientNetworkAnimator.cs

```diff
@@ -0,0 +1,115 @@
+using System.Collections;
```

Comment:
This sample is 3 things:
- caching client IDs that are not the owner for perf reasons
- overriding anim state broadcasting using those client IDs
- overriding trigger broadcasting using those client IDs